sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -321,7 +320,13 @@ describe("Test createFromAadTokenCredentials", function(): void {
     "Endpoint=sb://((.*).servicebus.windows.net)"
   ) || "")[1];
 
-  async function testCreateFromAadTokenCredentials(host: string, tokenCreds: any): Promise<void> {
+  async function testCreateFromAadTokenCredentials(host: any, tokenCreds?: any): Promise<void> {
+    if (tokenCreds == undefined) {
+      // For the Node only tests that rely on AAD, we make sure to not pass in the tokenCreds
+      // so we can obtain them from AAD
```

Comment:
The `Coerces input to string for host in createFromAadTokenCredentials` test case is only for node, but there we run `getTokenCredentialsFromAAD()` to get the token creds and pass it to `testCreateFromAadTokenCredentials()`.

The `sends a message to the ServiceBus entity` test case also falls under the same bucket, but there we don't pass in tokenCreds and this code here uses the creds from  `getTokenCredentialsFromAAD()` 

Based on the current state of the PR, there doesnt seem to be a need for the `tokenCreds` to be an optional parameter to `testCreateFromAadTokenCredentials()`

Reply:
Reverted

Reply:
Now that `tokenCreds` is always passed, you can also remove the code here that calls `getTokenCredentialsFromAAD()`.

Reply:
Updated